src/modelframe.jl

```diff
@@ -239,7 +240,8 @@ function coefnames(mf::ModelFrame)
     for (i_term, term) in enumerate(terms.terms)
 
         ## names for columns for eval terms
-        names = Vector{String}[]
+    
```

Comment:
Remove this.

Reply:
Change back `Vector{Vector{String}}()` for `Vector{String}[]`?

Reply:
Sorry, I meant the new empty line. Same at line 144.